src/main/java/com/matera/cadastrocentral/authentication/AuthenticationService.java

```diff
@@ -35,4 +34,22 @@ public void changePassword(UUID clientId, PasswordDTO newPassword) {
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Insira uma senha numérica de 6 digítos");
         }
     }
+
+    public void loginValidation(LoginDTO loginDTO) {
+        Client client = clientService.getClientByCPF(loginDTO.getClientCPF());
+        if(!client.getPassword().equals(loginDTO.getClientPassword())){
+            throw new ResponseStatusException(
+                    HttpStatus.UNAUTHORIZED,
+                    "Password incorrect!");
+        }
+    }
+
+    public void passwordValidation(UUID clientId, PasswordDTO clientPassword) {
+        Optional<Client> client = clientService.getClientById(clientId);
+        if (!client.get().getPassword().equals(clientPassword.getPassword())){
+            throw new ResponseStatusException(
+                    HttpStatus.FORBIDDEN,
+                    "Password incorrect!");
```

Comment:
Maybe change messages to PTBR, because all the other ones are in portuguese

Reply:
When do we use passwordValidation? If we use only when checking if two input passwords are the same, I guess we could change the message to passwords don't match

Reply:
I was thinking of using this validation for cases where the user is logged in and needs to enter the password, such as when transferring and deleting the account.

The validation if two passwords are the same I do not know how it will be done, maybe straight from the front?